examples/addition.rs

```diff
@@ -54,15 +54,15 @@ fn run() -> Result<(), Box<Error>> {
     graph.import_graph_def(&proto, &ImportGraphDefOptions::new())?;
     let mut session = Session::new(&SessionOptions::new(), &graph)?;
 
-    // Run the Step
-    let mut step = StepWithGraph::new();
-    step.add_input(&graph.operation_by_name_required("x")?, 0, &x);
-    step.add_input(&graph.operation_by_name_required("y")?, 0, &y);
-    let z = step.request_output(&graph.operation_by_name_required("z")?, 0);
-    session.run(&mut step)?;
+    // Run the graph.
+    let mut args = SessionRunArgs::new();
+    args.add_input(&graph.operation_by_name_required("x")?, 0, &x);
```

Comment:
(optional) inputs/outputs are named feeds and fetches in TensorFlow nomenclature as related to session runs. Either way works, though.

Reply:
Yeah, perhaps rename them. Up to you.